pandas/tests/arithmetic/test_period.py

```diff
@@ -16,6 +16,7 @@
 from pandas.core import ops
 from pandas import Period, PeriodIndex, period_range, Series
 from pandas.tseries.frequencies import to_offset
```

Comment:
so we can't do this, instead move the tick_classes fixture to pandas/conftest.py. I *think* everything should still work

Reply:
done.